test/e2e_node/jenkins/image-config.yaml

```diff
@@ -8,9 +8,10 @@ images:
   ubuntu-docker10:
     image: e2e-node-ubuntu-trusty-docker10-v1-image
     project: kubernetes-node-e2e-images
-  coreos-stable:
-    image: e2e-node-coreos-alpha-1068-20160707-image
-    project: kubernetes-node-e2e-images
+  coreos-alpha:
```

Comment:
Stability can mean a lot of things. For the purpose of these tests I think it's a good choice and if we know of any issues with its systemd or kubelet version we can hold off on updating until we're ready for it.